tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -571,3 +572,240 @@ def test_invalid_interval(self):
+                    "interval": "1h",
+                    "yAxis": "count()",
+                    "field": ["message", "user.email"],
+                    "topEvents": [self.event1_slug, self.event2_slug, self.event3_slug],
+                },
+                format="json",
+            )
+
+        data = response.data
+        assert response.status_code == 200, response.content
+        assert len(data) == 3
+
+        event1_data = data[self.event1_slug]
+        assert event1_data["values"] == {"message": "poof", "email": self.user.email}
+        assert [attrs for time, attrs in event1_data["data"]] == [[{"count": 3}], [{"count": 0}]]
+
+        event2_data = data[self.event2_slug]
+        assert event2_data["values"] == {"message": "voof", "email": self.user2.email}
+        assert [attrs for time, attrs in event2_data["data"]] == [[{"count": 0}], [{"count": 3}]]
+
+        event3_data = data[self.event3_slug]
+        assert event3_data["values"] == {"message": "very bad", "email": "foo@example.com"}
+        assert [attrs for time, attrs in event3_data["data"]] == [[{"count": 3}], [{"count": 3}]]
+
+    def test_top_events_with_projects(self):
+        with self.feature("organizations:discover-basic"):
+            response = self.client.get(
+                self.url,
+                data={
+                    "start": iso_format(self.day_ago),
+                    "end": iso_format(self.day_ago + timedelta(hours=1, minutes=59)),
+                    "interval": "1h",
+                    "yAxis": "count()",
+                    "field": ["message", "project"],
+                    "topEvents": [self.event1_slug, self.event2_slug],
+                },
+                format="json",
+            )
+
+        data = response.data
+
+        assert response.status_code == 200, response.content
+        assert len(data) == 2
+
+        event1_data = data[self.event1_slug]
+        values = event1_data["values"]
+        assert values["message"] == "poof"
+        assert values["project"] == self.project.slug
+        assert [attrs for time, attrs in event1_data["data"]] == [[{"count": 3}], [{"count": 0}]]
+
+        event2_data = data[self.event2_slug]
+        values = event2_data["values"]
+        assert values["message"] == "voof"
+        assert values["project"] == self.project2.slug
+        assert [attrs for time, attrs in event2_data["data"]] == [[{"count": 0}], [{"count": 3}]]
+
+    def test_top_events_with_issue(self):
+        with self.feature("organizations:discover-basic"):
+            response = self.client.get(
+                self.url,
+                data={
+                    "start": iso_format(self.day_ago),
+                    "end": iso_format(self.day_ago + timedelta(hours=1, minutes=59)),
+                    "interval": "1h",
+                    "yAxis": "count()",
+                    "field": ["message", "issue.id"],
```

Comment:
This should be `"issue"` not `"issue.id"` I think. At least, the user can't send `issue.id`. Maybe the frontend overrides that.

Reply:
Users can send issue.id if they know about it. But the UI only exposes `issue`.